tgui/packages/tgui/interfaces/ForbiddenLore.js

```diff
@@ -0,0 +1,59 @@
+import { useBackend } from '../backend';
+import { map } from 'common/collections';
+import { Button, Section, Box, Tabs } from '../components';
+import { Window } from '../layouts';
+
+export const ForbiddenLore = (props, context) => {
+  const { act, data } = useBackend(context);
+  // Extract `health` and `color` variables from the `data` object.
+  const {
+    charges,
+    to_know = {},
+  } = data;
+  return (
+    <Window resizable>
+      <Window.Content scrollable>
+        <Section title="Research Eldritch Knowledge">
+          Charges left : {charges}
+          {to_know !== null ? (
+            to_know.map(knowledge => (
+              <Section
+                key={knowledge.name}
+                title={knowledge.name}
+
+                level={2}>
+                <Box>
+                  <b> {knowledge.path} path</b>
+                </Box>
+                <br />
+                <Box>
+                  <Button
+                    content={knowledge.state}
+                    disabled={knowledge.disabled}
+                    onClick={() => act('research', {
+                      name: knowledge.name,
+                      cost: knowledge.cost,
+                    })} />
+                  {' '}
+                  Cost : {knowledge.cost}
+                </Box>
+                <br />
```

Comment:
Don't use `<br />` for adding an extra space between the boxes, use `mb={1}` or `mt={1}` on boxes instead.